src/Helper/DbHelper.js

```diff
@@ -26,15 +26,15 @@ export const getSubscriptionStatusDB = async () => {
 	var res = await localforage.getItem("platforms");
 	if (res === null)
 		return {
-			dailyChallenge: true,
```

Comment:
A small bug which let to error on subscribe page.
As 
![image](https://github.com/nisarg0/Kontest-Reminder/assets/60577767/a4b0578a-9aa4-4bd6-b2d5-989b6a22ccd9)

We were comparing "dailyChallenge" and not "dailycallenge". Fixed this.